ui-wcm-admin-smoke/src/main/java/com/adobe/cq/cloud/testing/it/ui/wcm/admin/smoke/AssetsAndFilesActionIT.java

```diff
@@ -0,0 +1,331 @@
+/*
+ * Copyright 2019 Adobe Systems Incorporated
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.adobe.cq.cloud.testing.it.ui.wcm.admin.smoke;
+
+import com.adobe.cq.cloud.testing.it.ui.wcm.admin.smoke.rules.CleanUpRule;
+import com.adobe.cq.cloud.testing.it.ui.wcm.admin.smoke.rules.TempFolderRule;
+import com.adobe.cq.testing.client.CQClient;
+import com.adobe.cq.testing.client.JsonClient;
+import com.adobe.cq.testing.junit.rules.CQAuthorPublishClassRule;
+import com.adobe.cq.testing.junit.rules.CQRule;
+import com.adobe.cq.testing.util.TestUtil;
+import org.apache.sling.testing.clients.ClientException;
+import org.apache.sling.testing.clients.util.ResourceUtil;
+import org.apache.sling.testing.clients.util.poller.Polling;
+import org.codehaus.jackson.JsonNode;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.RuleChain;
+import org.junit.rules.TestRule;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.concurrent.TimeoutException;
+
+/**
+ * A set of simple asset tests to verify that asset creating is working.
+ */
+public class AssetsAndFilesActionIT {
```

Comment:
I would tend to leave this one outside of the Sites Admin UI functionality; sounds rather like DAM UI